esphome/components/kalman_combinator/kalman_combinator.cpp

```diff
@@ -0,0 +1,104 @@
+#include "kalman_combinator.h"
+#include "esphome/core/hal.h"
+#include <cmath>
+#include <functional>
+
+namespace esphome {
+namespace kalman_combinator {
+
+void KalmanCombinatorComponent::dump_config() {
+  ESP_LOGCONFIG("kalman_combinator", "Kalman Combinator:");
+  ESP_LOGCONFIG("kalman_combinator", "  Update variance per ms: %f", this->update_variance_);
+  ESP_LOGCONFIG("kalman_combinator", "  Sensors:");
+  for (const auto &sensor : this->sensors_) {
+    auto &entity = *sensor.first;
+    ESP_LOGCONFIG("kalman_combinator", "    - %s", entity.get_object_id().c_str());
```

Comment:
I think using the name here is much clearer.
```suggestion
    ESP_LOGCONFIG("kalman_combinator", "    - %s", entity.get_name().c_str());
```